openfisca_uk_data/datasets/frs/frs_was_imputation.py

```diff
@@ -0,0 +1,191 @@
+from openfisca_uk_data.utils import dataset, UK
+from openfisca_uk_data.datasets.frs.frs import FRS
+import pandas as pd
+import microdf as mdf
+import sklearn
+import synthimpute as si
+import h5py
+import numpy as np
+
+
+@dataset
+class FRS_WAS_Imputation:
```

Comment:
Should we call this something else? I think we'd ideally make a master FRS-based dataset with WAS imputations, SPI adjustments, LCF imputations, etc.

Reply:
In future yes, OK to merge for now though?